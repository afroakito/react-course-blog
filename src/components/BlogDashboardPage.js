import React from 'react';
import BlogList from './BlogList';
import BlogListFilters from './BlogListFilters';

const BlogDashboardPage = () => (
  <div>
    <BlogListFilters />
    <BlogList />
  </div>
);

export default BlogDashboardPage;
