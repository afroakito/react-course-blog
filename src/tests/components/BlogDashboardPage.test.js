import React from 'react';
import { shallow } from 'enzyme';
import BlogDashboardPage from '../../components/BlogDashboardPage';

test('should render BlogDashboardPage correctly', () => {
  const wrapper = shallow(<BlogDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
