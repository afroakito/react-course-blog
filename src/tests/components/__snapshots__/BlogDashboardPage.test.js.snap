// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BlogDashboardPage correctly 1`] = `
<div>
  <BlogListFilters />
  <Connect(BlogList) />
</div>
`;
